### ISSUE TITLE
Seperate option to change height of emoji panel

### ISSUE THREAD

**Issue by imDusht** [florisboard/florisboard#1143]
Add an option to change emoji panel height seperately. The issue is that i have set my keyboard height very short and it works fine for me but at the same time emoji panel shows shows just 2 line of emojis which isn't convenient for a long list to scroll.
Thanks (Screenshots attached).
![Screenshot_2021-08-08-16-22-28-542](https://user-images.githubusercontent.com/79660867/128630629-ee8d0db8-d769-41d3-8616-93c5e9b53acb.jpeg)
![Screenshot_2021-08-08-16-22-40-014](https://user-images.githubusercontent.com/79660867/128630644-969ba2dd-8bad-4a24-a392-fd2d3514c45f.jpeg)

**Comment by patrickgold** [comment]
I agree, the emoji list looks cut off when using "very short" as the height. Will add this issue to the todo list for the emoji rework.

**Comment by lm41** [comment]
The keyboard sizing was reworked with `v0.6.0-alpha01`, but it is better to test with the latest debug build (https://github.com/florisboard/florisboard/actions/runs/21232604406) as there were some sizing changes after the release. It should feel "more organic" with the update. Can you try out the debug and preview version and let us know if the problem was resolved?
